e2e/testdata/fn-render/basicpipeline-v1alpha1/.expected/config.yaml

```diff
@@ -0,0 +1,14 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+exitCode: 1
```

Comment:
What's this testing? That we don't support v1alpha1? Can you assert an error message to make that clear?

Reply:
Yes. I have added message assertion.